package/Linux/gateway/debian/postinst

```diff
@@ -3,14 +3,18 @@
 # System-wide package configuration.
 DEFAULTS_FILE="/etc/default/devolutions-gateway"
 
+if [ ! -f $DEFAULTS_FILE ]; then
+	/bin/touch $DEFAULTS_FILE
+fi
+
 if [ ! -d /etc/devolutions-gateway ]; then
 	/bin/mkdir /etc/devolutions-gateway
 	/bin/chmod 655 /etc/devolutions-gateway
 fi
 
 if [ -d /run/systemd/system ]; then
 	/usr/bin/devolutions-gateway service register >/dev/null
-	/usr/bin/devolutions-gateway service --config-init-only >/dev/null
-	systemctl restart devolutions-gateway >/dev/null 2>&1
 	systemctl daemon-reload
+	/usr/bin/devolutions-gateway --config-init-only >/dev/null
+	systemctl restart devolutions-agent >/dev/null 2>&1
```

Comment:
This is a pertinent improvement, the correct command is `/usr/bin/devolutions-gateway --config-init-only`.
(But we should not be using the name `devolutions-agent`, the fix is not to change that here.)